packages/nextjs/app/dashboard/reclaim/page.tsx

```diff
@@ -1,14 +1,39 @@
 "use client";
 
-import React, { useState } from "react";
+import React, { useEffect, useMemo, useState } from "react";
 import { CiFilter } from "react-icons/ci";
 import { GoArrowUpRight } from "react-icons/go";
 import Button from "~~/components/Button/Button";
 import FilterProviderModal from "~~/components/FilterProviderModal/FIlterProviderModal";
 
 const Reclaim: React.FC = () => {
+  const providerData = useMemo(
+    () => [
+      {
+        providerName: "Binance",

```

Comment:
is this filtering only Binance and OKX ?

Reply:
It should filter LinkedIn also and Github and X analytics. 

Change Description into KYC for OKX and Binance and into Dashboard Analytics for LinkedIn  and into Developer stats for Github.

Reply:
Yes because the mock data i created only have binance and okx but the filter can filter all the provider